sdk/search/Azure.Search.Documents/src/Pipeline/OdataMetadataPolicy.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Search.Documents.Pipeline
```

Comment:
Let's not introduce a new namespace.  It might still show up in various tooling even though there aren't any public types.

Reply:
Sorry, meant to remove that. VS adds it by default, making Search's subfolders a PITA (we used that as-is for KV).

Reply:
If transforming via autorest.md works, this whole file will go away anyway.